statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -378,15 +382,15 @@ def test_acorr(self):
         acorrs = self.res.acorr(10)
 
     def test_forecast(self):
-        point = self.res.forecast(self.res.y[-5:], 5)
+        point = self.res.forecast(self.res.endog[-5:], 5)
```

Comment:
Very much +1 for getting rid of the multiple aliases with the `# keep alias for now`s

Reply:
https://github.com/jbrockmendel/sm2/blob/master/sm2/tools/decorators.py#L12

Reply:
I added this decorator with some extensions as part of this PR.  Saved a lot of error prone copy-paste pain upgrading attributes to properties.

Reply:
I like the edits you made, will incorporate them downstream.  There's also a test ready to be copied.